addons/sourcemod/scripting/szf/config.sp

```diff
@@ -1,5 +1,6 @@
-#define CONFIG_WEAPONS "configs/szf/weapons.cfg"
-#define CONFIG_CLASSES "configs/szf/classes.cfg"
+#define CONFIG_WEAPONS       "configs/szf/weapons.cfg"
+#define CONFIG_CLASSES       "configs/szf/classes.cfg"
+#define CONFIG_WEAPONINDEXES "configs/szf/weaponindexes.cfg"
```

Comment:
`reskins.cfg` or `weaponreskins.cfg` may be a better name when it only used to match reskins with original.